instrumentation/README.adoc

```diff
@@ -1,5 +1,13 @@
 :imagesdir: ../images
 
-This section describes how to instrument your applications to capture information that can be used to analyse their use.
+== Instrumenting an Application
+
+In order to get tracing data from your application it needs to be instrumented.
+
+There are more several supported ways how your applications can be instrumented:
+
+* Non-intrusive Java agent approach
+* Zipkin instrumentaiton libraries
+* Hawkular OpenTracing implementation (experimental)
 
```

Comment:
Should we also add 'Custom instrumentation reporting trace fragments to APM server' ?